update/application.go

```diff
@@ -141,6 +145,16 @@ func (cmd *applicationCmd) applyUpdates(app *apps.Application) {
 	if cmd.Env != nil {
```

Comment:
I know this has nothing to do with your changes, but this statement seems to be wrong? It should be `if cmd.BuildEnv != nil`, right?

Reply:
yeah. good catch! 👌